src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -88,6 +88,9 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "ReconstructedJets",
             "ReconstructedElectrons",
 
+	    // Tracks propagated to calorimeters
+	    "PropagatedTrackPoints",
```

Comment:
You would not need a comment if the name was self-explanatory. Maybe, "TrackCalorimeterProjections"?